experiments/openebs/openebs-pool-disk-loss/openebs_pool_disk_loss_ansible_logic.yml

```diff
@@ -0,0 +1,213 @@
+- hosts: localhost
+  connection: local
+
+  vars:
+    a_label: "{{ lookup('env','APP_LABEL') }}"
+    a_ns: "{{ lookup('env','APP_NAMESPACE') }}"
+    a_pvc: "{{ lookup('env','APP_PVC') }}"
+    c_duration: "{{ lookup('env','TOTAL_CHAOS_DURATION') }}"
+    c_experiment: "openebs-pool-disk-loss"
+    ramp_time: "{{ lookup('env','RAMP_TIME') }}"
+    data_persistence: "{{ lookup('env','DATA_PERSISTENCE') }}"
+    liveness_label: "{{ lookup('env','LIVENESS_APP_LABEL') }}"
+    liveness_namespace: "{{ lookup('env','LIVENESS_APP_NAMESPACE') }}"
+    openebs_ns: "{{ lookup('env','OPENEBS_NAMESPACE') }}"
+    cloud_platform: "{{ lookup('env','CLOUD_PLATFORM') }}"
+    c_ns: "{{ lookup('env','CHAOS_NAMESPACE') }}"
+    disk_name: "{{ lookup('env','DISK_NAME') }}"
+    device_name: "{{ lookup('env', 'DEVICE_NAME') }}"
+    node_name: "{{ lookup('env','NODE_NAME') }}"
+    project_id: "{{ lookup('env','PROJECT_ID') }}"
+    zone_name: "{{ lookup('env','ZONE_NAME') }}"
+    auxiliary_appinfo: "{{ lookup('env','AUXILIARY_APPINFO') }}"
+
+  vars_files:
+    - /mnt/parameters.yml
+    - /experiments/openebs/openebs_components.yml
+  
+  tasks:
+    - block:
+
+      - include_tasks: /utils/runtime/create_testname.yml  
+
+      - include_tasks: openebs_pool_disk_loss_ansible_prerequisites.yml
+
+      # GENERATE EXPERIMENT RESULT NAME AND CREATE CHAOS RESULT RESOURCE
+      - block:
```

Comment:
This block should container only Construct chaos result name (experiment_name) and executed only if this condition (when: lookup('env','CHAOSENGINE')) passed